components/chainhook-cli/src/service/tests/mod.rs

```diff
@@ -1,7 +1,5 @@
-use chainhook_sdk::chainhooks::types::{
-    ChainhookFullSpecification, ChainhookSpecification, StacksChainhookFullSpecification,
-};
-use chainhook_sdk::types::{Chain, StacksNetwork};
+use chainhook_sdk::chainhooks::types::ChainhookFullSpecification;
```

Comment:
A lot of the changes in this file are more cleanup:
 - moving out helpers
 - our `mine_block` function now takes fork ids, so updating existing calls to that function to pass an fork id
 - our helpers have a new struct for the return value, so updating the tests that use that helper
 - simplifying our code to clean up temp test data

The actual change is the addition of the `it_saves_unconfirmed_blocks` test all the way at the bottom